sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/GpuHashJoin.scala

```diff
@@ -601,6 +601,14 @@ class HashFullJoinIterator(
       try {
         val lazyLeftMap = LazySpillableGatherMap(maps(0), spillCallback, "left_map")
         val lazyRightMap = LazySpillableGatherMap(maps(1), spillCallback, "right_map")
+        withResource(new NvtxWithMetrics("update tracking mask",
+          NvtxColor.ORANGE, joinTime)) { _ =>
+          closeOnExcept(lazyLeftMap) { lazyLeft =>
+            closeOnExcept(lazyRightMap) { lazyRight =>
+              updateTrackingMask(if (buildSide == GpuBuildRight) lazyRight else lazyLeft)
+            }
+          }
```

Comment:
nit: could be slightly more compact
```suggestion
          closeOnExcept(Seq(lazyLeftMap, lazyRightMap)) { _ =>
            updateTrackingMask(if (buildSide == GpuBuildRight) lazyRightMap else lazyLeftMap)
          }
```

Reply:
Thanks for the suggestion.  I will update.